docs/pipelines/caching/index.md

```diff
@@ -88,6 +88,36 @@ In this example, the cache key contains three parts: a static string ("yarn"), t
 
 On the first run after the task is added, the cache step will report a "cache miss" since the cache identified by this key does not exist. After the last step, a cache will be created from the files in `$(Pipeline.Workspace)/.yarn` and uploaded. On the next run, the cache step will report a "cache hit" and the contents of the cache will be downloaded and restored.
 
+#### Restore keys
+
+`restoreKeys` can be used if one wants to query against multiple exact keys or key prefixes. This is used to fallback to another key in the case that a `key` does not yield a hit. A restore key will search for a key by prefix and yield the latest created cache entry as a result. This is useful if the pipeline is unable to find an exact match but wants to use a partial cache hit instead. To insert multiple restore keys, simply delimit them by using a new line and `-` character for indicate the restore key (see the example for more details). The order of which restore keys will be tried against will be from top to bottom.
+
+#### Example
+
+Here is an example on how to use restore keys by Yarn:
+
+```yaml
+variables:
+  YARN_CACHE_FOLDER: $(Pipeline.Workspace)/.yarn
+
+steps:
+- task: CacheBeta@0azure pipelines 
+  inputs:
+    key: yarn | $(Agent.OS) | yarn.lock
+    path: $(YARN_CACHE_FOLDER)
+    restoreKeys:
+      - yarn | $(Agent.OS)
+      - yarn
```

Comment:
I think in yaml it should be:
restoreKeys: |
    yarn | $(Agent.OS)
    yarn

Reply:
yeah i don't think there is a dash

Reply:
Changed to new line delimiter